src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -90,6 +90,7 @@ class IMain(val settings: Settings, parentClassLoaderOverride: Option[ClassLoade
     *  on the future.
     */
   private var _classLoader: AbstractFileClassLoader = null  // active classloader
+  private var _runtimeMirror: reflect.runtime.universe.Mirror = null
```

Comment:
There's `ru`m for improvement.

Reply:
rum would be an improvement right about now.